xml/deploy_cephadm.xml

```diff
@@ -1435,7 +1443,11 @@ spec:
   pool: iscsi-pool
   api_user: iscsi-user
   api_password: password
+  trusted_ip_list: "<replaceable>IP_ADDRESS</replaceable>, <replaceable>IP_ADDRESS</replaceable>"
```

Comment:
`IP_ADDRESS_1, IP_ADDRESS_2` instead of `IP_ADDRESS, IP_ADDRESS`?

Reply:
The presumption is the reader is smart enough to assertain to include the same IP address twice?